go.mod

```diff
@@ -4,36 +4,26 @@ go 1.15
 
 require (
 	github.com/go-git/go-git/v5 v5.2.0
-	github.com/go-logr/logr v0.2.0
-	github.com/go-logr/zapr v0.1.1 // indirect
-	github.com/go-openapi/spec v0.19.6
-	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/gophercloud/gophercloud v0.6.0 // indirect
+	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/zapr v0.4.0 // indirect
+	github.com/go-openapi/spec v0.20.2
 	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/gomega v1.10.3
 	github.com/prometheus/client_golang v1.8.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/pipeline v0.20.1
+	github.com/tektoncd/pipeline v0.21.0
 	go.uber.org/zap v1.16.0
-	k8s.io/api v0.18.12
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/code-generator v0.18.12
-	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
-	k8s.io/kubectl v0.18.10
-	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
-	knative.dev/pkg v0.0.0-20210107022335-51c72e24c179
+	k8s.io/api v0.19.7
+	k8s.io/apimachinery v0.19.7
+	k8s.io/client-go v0.19.7
+	k8s.io/code-generator v0.19.7
```

Comment:
This one call my attention, because this one is used in [here](https://github.com/shipwright-io/build/blob/master/pkg/kubecodegen/set-kubecodegen-import.go#L13) so that we can use the script inside the vendor in this other  [update-codegen.sh](https://github.com/shipwright-io/build/blob/master/hack/update-codegen.sh) script.

When I ran the [update-codegen.sh](https://github.com/shipwright-io/build/blob/master/hack/update-codegen.sh), it seems that the new version(v0.19.7) is adding new entries on the generated files, as seen in my local git diff:

```sh
$ git status
HEAD detached at zoe/bump-tekton-to-v0.21.0
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/client/clientset/versioned/fake/register.go
	modified:   pkg/client/listers/build/v1alpha1/build.go
	modified:   pkg/client/listers/build/v1alpha1/buildrun.go
	modified:   pkg/client/listers/build/v1alpha1/buildstrategy.go
	modified:   pkg/client/listers/build/v1alpha1/clusterbuildstrategy.go
```

can you pls make sure we get also an update for this. In general, I will recommend you to run all of the other make targets that might generate something.

Reply:
Thanks @qu1queee ! Now, I have run `hack/update-codegen.sh`, `hack/generate-fakes.sh` and `hack/generate-copyright.sh` to generate new things. Also run other checks locally.